File: airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/exception/DestinationException.java
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.workers.internal.exception;

/**
 * Exceptions thrown from a destination.
 */
public class DestinationException extends RuntimeException {

  public DestinationException(final String message) {
    super(message);
  }

  public DestinationException(final String message, final Throwable cause) {
    super(message, cause);
  }

}

File: airbyte-commons-worker/src/main/kotlin/io/airbyte/workers/internal/exception/DestinationException.kt
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.workers.internal.exception

/**
 * Exceptions thrown from a destination.
 */
class DestinationException : RuntimeException {
  constructor(message: String?) : super(message)

  constructor(message: String?, cause: Throwable?) : super(message, cause)
}
